corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -748,7 +748,7 @@ def javascript_libraries(context, **kwargs):
     return {
         'request': getattr(context, 'request', None),
         'underscore': kwargs.pop('underscore', False),
-        'jquery': kwargs.pop('jquery', False),
```

Comment:
Looks like `jquery` was never used. Do you know why it was here before?

Reply:
Originally it was an option, then I removed it in https://github.com/dimagi/commcare-hq/pull/18745/commits/72d963c1a8ade544268377355f037f4b1993d9aa but forgot to update this file.